mage_ai/orchestration/pipeline_scheduler.py

```diff
@@ -251,6 +267,15 @@ def schedule(self, block_runs: List[BlockRun] = None) -> None:
                 self.pipeline_run.update(
                     status=PipelineRun.PipelineRunStatus.FAILED)
 
+                # Backfill status updated to "failed" if at least 1 of its pipeline runs failed
+                if backfill is not None and any(
+                    [PipelineRun.PipelineRunStatus.FAILED == pr.status
+                        for pr in backfill.pipeline_runs]
```

Comment:
should this be `backfill.pipeline_runs` or `latest_pipeline_runs`?

Reply:
It should be `latest_pipeline_runs`. Thanks for catching that. It was still working correctly when I tested it since that `if` block checks for any failed blocks in the current pipeline run, but there could be some edge cases it wouldn't have covered. Updated.